GRIB2_Template_3_101_GridDefinitionTemplate_en.csv

```diff
@@ -1,5 +1,5 @@
-Title_en,OctetNo,Contents_en,Note_en,Status
-general unstructured grid,15,Shape of the Earth,(see Code table 3.2),Operational
-general unstructured grid,16-18,Number of grid used (defined by originating centre),,Operational
-general unstructured grid,19,Number of grid in reference (to allow annotating for Arakawa C-grid on arbitrary grid),(see Note),Operational
-general unstructured grid,20-35,Universally Unique Identifier of horizontal grid,,Operational
+Title_en,OctetNo,Contents_en,Note_en,noteIDs,codeTable,flagTable,Status
+general unstructured grid,15,Shape of the Earth,(see Code table 3.2),,3.2,,Operational
+general unstructured grid,16-18,Number of grid used (defined by originating centre),,,,,Operational
+general unstructured grid,19,Number of grid in reference (to allow annotating for Arakawa C-grid on arbitrary grid),(see Note),86,,,Operational
```

Comment:
@amilan17 should it be (see Note 86)?

Reply:
@amilan17 should it be" (see Note 1), 86,,," . I notice that normally the note in the template starts from 1.

Reply:
leaving as is. As discussed, when there is only one note, no number is assigned.